### DISCUSSION TITLE
[Bug/Feature request] 'Stars' section: Add page numbers

### DISCUSSION THREAD

**Discussion by d-kleine** [community/community#117793] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In absence of a designated 'Stars' section within the community feedback, I'm sharing this input under 'Lists':
![grafik](https://github.com/community/community/assets/53251018/c258f2bf-f41f-4bd2-a87b-9c89026a496b)

I'm uncertain whether this is a bug or a missing feature, but when the number of starred repositories exceeds 30, there's no indication of the current page or the total number of pages in the 'Stars' section. Currently, only 'Previous' and 'Next' buttons are provided:
![grafik](https://github.com/community/community/assets/53251018/5e6fb833-f123-4606-8e41-b785e3f22193)

From a usability perspective, it would greatly enhance user experience to include information such as '_Page [current page number] of [total number of pages]_' at the end of the 'Stars' section. Additionally, renaming the 'Previous' button to 'Previous page' and 'Next' button to 'Next page' would clarify their functions.

**Comment by godcodehunter** [comment]
Bump
